### PR TITLE
Add new News colours

### PR DESCRIPTION
**Pull request by greenc05**
Adding new colours as part of the following ticket

JIRA
https://jira.dev.bbc.co.uk/browse/NEWS-5323

### CODE REVIEW

**Comment by greenc05** [comment]
Any chance of a review when you have a quick sec please @shaunbent?

**Comment by shaunbent** [comment]
Looks good to me. You will need to bump the `bower.json`.

Are you ok with the `$nw-` prefix or do you want to use `$news-`?!?

**Comment by greenc05** [comment]
Great, thanks @shaunbent Have just bumped the bower.json

We'll get used to the prefix, and it follows what you've done so I think lets stick with it, thanks.

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "gs-sass-tools",
-  "version": "4.0.1",
+  "version": "4.0.2",
```
Comment (shaunbent) [inline bower.json]: I would suggest this is a minor change as you're adding new functionality.

**Comment by shaunbent** [comment]
Merge away when you've resolved the conflicts.

**Comment by greenc05** [comment]
Thanks @shaunbent 

Conflicts resolved and merged, what's the deployment process?

**Comment by shaunbent** [comment]
You need to create a 'Release' in GitHub, I've just done this for you: https://github.com/bbc/gs-sass-tools/releases/tag/4.1.1

In your component you just need to run `bower install` and it should pull down this version

**Comment by greenc05** [comment]
Super thank you @shaunbent

**Pull request merged**